test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -267,7 +319,7 @@ private class GbmExample
             public float[] Score;
         }
 
-        private void LightGbmHelper(bool useSoftmax, out string modelString, out List<GbmExample> mlnetPredictions, out double[] lgbmRawScores, out double[] lgbmProbabilities)
+        private void LightGbmHelper(bool useSoftmax, out string modelString, out List<GbmExample> mlnetPredictions, out double[] lgbmRawScores, out double[] lgbmProbabilities, double sigmoid = .5)
```

Comment:
The parameter `sigmoid` can appear before those `out` variables.

Reply:
Since `sigmoid` is an optional parameter it has to come after all the required parameters, including the out parameters. I could refactor the code to make it not optional and just pass in the default value from the other methods, but that is the only way to have it come before the `out` params in the list.

Reply:
Explicitly specify it to 0.5 for all default cases can solve your concern maybe? As a test helper, I feel exposing all information explicitly is useful.

Reply:
I changed it so its no longer a default value and is explicitly passed in from each method and moved it before the out params.